docs/cow-protocol/tutorials/solvers/cow-amm.md

```diff
@@ -63,35 +38,35 @@ You need to choose a valid CoW Swap order with the following restrictions:
 - `sellTokenBalance`: must be `BALANCE_ERC20`.
 - `buyTokenBalance`: must be `BALANCE_ERC20`.
 
+:::note
+
+The sell and buy amount specified are the limit amounts of the order. The actual executed amounts are determined using prices from the auction.
+In that regard, CoW AMM orders behave exactly like normal trader orders.
```

Comment:
Can we add that if solvers are using non co-located drivers, they should encode this trade as a trade of `kind=jit` (specifying executed amount and fee amount)?

Reply:
I added a reference to the API specification for jit trades and and preinteractions at the bottom, after the discussion of `commit`.